app/models/hackbot/interactions/lookup.rb

```diff
@@ -0,0 +1,125 @@
+module Hackbot
+  module Interactions
+    class Lookup < AdminCommand
+      include ActionView::Helpers::DateHelper
+
+      TRIGGER = /lookup <@(?<uid>.*)>/
+
+      # Quick heads up that &lt; and &gt; are the HTML encodings for < and >,
+      # respectively. Slack requires these characters be HTML encoded.
+      USAGE = 'lookup &lt;@username&gt;'.freeze
```

Comment:
This looks different than https://github.com/hackclub/api/blob/master/app/models/hackbot/interactions/dice_roll.rb#L6